Packs/CommonWidgets/Scripts/MyToDoTasksWidget/MyToDoTasksWidget.py

```diff
@@ -47,10 +49,13 @@ def get_open_to_do_tasks_of_current_user() -> List[Dict]:
     return table
 
 
-def main():
+def main():  # pragma: no cover
     try:
         results = get_open_to_do_tasks_of_current_user()
-        table_name = f'My ToDo Tasks ({len(results)})'
+        if len(results) >= MAX_ENTRIES:
+            table_name = f'My ToDo Tasks (First {MAX_ENTRIES} Tasks)'
```

Comment:
Can a user query the rest separately?

Reply:
Currently No.
Presenting `Task Data Source` in `Table` form is not supported at the moment (will open a feature request for that).
So this script is a work around to allow generating a table for Tasks in the UI.
The table is being generated inside this script as a README string, therefore, pagination is not an option.
Also, we need to limit the amount of entries displayed at any moment in the widget because of UI limitations.